### ISSUE TITLE
SyntaxError: invalid syntax

### ISSUE THREAD

**Issue by Yanjiayork** [parrt/random-forest-importances#32]
When import rfpimp, there is an error like below

"
**File "/Users/yan/anaconda/lib/python3.5/site-packages/rfpimp.py", line 518
    ax.xaxis.set_major_formatter(FormatStrFormatter(f'%.{xtick_precision}f'))
                                                                          ^
SyntaxError: invalid syntax**
"

**Comment by parrt** [comment]
Hmm...seems like Python2 is trying to execute that code.

**Comment by Yanjiayork** [comment]
Right. Shouldn't be as it clearly shows that its python3.5.

**Comment by parrt** [comment]
Yep,I saw that but only reality matters, not what we think.  Try specifically saying python3 and see if that changes anything. That error is from Python2 as it is certainly valid Python3 if I remember correctly, since I use that code frequently.

**Comment by Yanjiayork** [comment]
Thanks, Terence. Just tried python3 rfpimp.py, it still shows the error. Very strange, see below.

Yans-MBP:Desktop yan$ python3 rfpimp.py
  File "rfpimp.py", line 478
    ax.xaxis.set_major_formatter(FormatStrFormatter(f'%.{xtick_precision}f'))
                                                                          ^
SyntaxError: invalid syntax

**Comment by parrt** [comment]
wow! lemme try. no problem for me:

```
beast:master:~/github/random-forest-importances/src $ python3 rfpimp.py
beast:master:~/github/random-forest-importances/src $ python3
Python 3.7.4 (default, Aug 13 2019, 15:17:50) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> 
```

What is rfpimp.py?  Is that my file or did you create a file with the same name as my package? If you did, that is likely confusing Python.

**Comment by Yanjiayork** [comment]
Yeah. That's true. Although I put it on a different folder. I also changed the name and retried again. It still shows the same error. Very weird. Sign.

**Comment by parrt** [comment]
maybe `f'...'` is python 3.6 not 3.5?

**Comment by parrt** [comment]
Ah: "Python f-string is the newest Python syntax to do string formatting. It is available since Python 3.6."

**Comment by parrt** [comment]
My python setup.py says `    python_requires='>=3.6'` so you must have installed withA different version of Python.

**Comment by Yanjiayork** [comment]
Thanks. Just updated the version. It works now. Thank you very much.
